environment.yml

```diff
@@ -46,3 +46,16 @@ dependencies:
 
     - pip:  
         - "--editable=.[conda_dev]"
+
+# If you are on Linux, have a NVIDIA GPU, and are getting the warning:
+
+# W tensorflow/stream_executor/platform/default/dso_loader.cc:64] Could not load dynamic 
+# library 'libcudart.so.11.0'; dlerror: libcudart.so.11.0: cannot open shared object 
+# file: No such file or directory
+
+# then activate the environment and run the commands:
+
+# mkdir -p $CONDA_PREFIX/etc/conda/activate.d
+# echo '#!/bin/sh' >> $CONDA_PREFIX/etc/conda/activate.d/libcudart_activate.sh
+# echo 'export LD_LIBRARY_PATH=$CONDA_PREFIX/lib:$LD_LIBRARY_PATH' >> $CONDA_PREFIX/etc/conda/activate.d/libcudart_activate.sh
+# source $CONDA_PREFIX/etc/conda/activate.d/libcudart_activate.sh
```

Comment:
Possibly move to docs?

Reply:
This is also in the docs under GPU support. If you dislike it here I can remove?

Reply:
Removed